### PR TITLE
ECIP-1094: Integration of VeriBlock Proof-of-Proof into Ethereum Classic

### PR DESCRIPTION
**Pull request by max-sanchez**
This ECIP proposes the integration of VeriBlock PoP security to protect Ethereum Classic from long-range 51% reorganizations and 51% attack censorship attacks against smart contract protocols.

### CODE REVIEW

File: ECIP-VeriBlock-PoP-51%-Protection.md [hunk 1]
```diff
@@ -0,0 +1,569 @@
+---
+lang: en
+ecip: ECIP ????
```
Comment (q9f) [inline ECIP-VeriBlock-PoP-51%-Protection.md:3]:
```suggestion
ecip: ECIP 1094
```

Comment (q9f) [reply ECIP-VeriBlock-PoP-51%-Protection.md:3]: please rename the file to ecip-1094.md

File: ECIP-VeriBlock-PoP-51%-Protection.md [hunk 2]
```diff
@@ -0,0 +1,569 @@
...
+title: Proof-of-Proof 51% Attack Prevention
```
Comment (q9f) [inline ECIP-VeriBlock-PoP-51%-Protection.md:4]:
```suggestion
title: VeriBlock Proof-of-Proof 51%-Attack Prevention
```

File: ECIP-VeriBlock-PoP-51%-Protection.md [hunk 3]
```diff
@@ -0,0 +1,569 @@
...
+status: Draft
+type: Standards Track
```
Comment (q9f) [inline ECIP-VeriBlock-PoP-51%-Protection.md:6]: please add `category: core`

File: ECIP-VeriBlock-PoP-51%-Protection.md [hunk 4]
```diff
@@ -0,0 +1,569 @@
...
+
+Proof-of-Proof doesn’t replace Ethereum Classic’s existing Proof-of-Work algorithm but provides reinforcement of it based on Bitcoin, and doesn’t introduce any centralized system or modify the blockchain’s governance system (adoption of hard forks, etc. is still done by consensus amongst PoW miners).
+
+Adjustments can be made to how Proof-of-Proof is integrated into Ethereum Classic based on tradeoffs between backwards compatibility with existing software, ease of implementation, data size, and tolerable levels of introduced complexity. As well, some parts of the VeriBlock integration can be skipped; the transaction offloading system that provides protection against 51% censorship attacks could be removed while still integrating Proof-of-Proof to provide security against surprise long-range reorganizations, and the integration of the `BITCOINFINALITY` opcode can be skipped to reduce the amount of data required to be stored directly on-chain (versus acquired by only nodes wanting to access Bitcoin finality statistics) if desired.
```
Comment (q9f) [inline ECIP-VeriBlock-PoP-51%-Protection.md:569]: Please add a license footer. See template for examples.

**Review by q9f — CHANGES_REQUESTED** [review]

**Comment by q9f** [comment]
Also, could you try to attend the next call? See #333

File: ECIP-VeriBlock-PoP-51%-Protection.md (hunk 4 above)
Comment (max-sanchez) [reply ECIP-VeriBlock-PoP-51%-Protection.md:570]: Added, thanks!

**Comment by q9f** [comment]
Can you change the filename to `ecip-1094.md` and the assets also?

File: ECIP-VeriBlock-PoP-51%-Protection.md [hunk 5]
```diff
@@ -0,0 +1,575 @@
+---
+lang: en
+ecip: ECIP 1094
```
Comment (q9f) [inline ECIP-VeriBlock-PoP-51%-Protection.md:3]:
```suggestion
ecip: 1094
```

File: ECIP-VeriBlock-PoP-51%-Protection.md [hunk 6]
```diff
@@ -0,0 +1,575 @@
...
+title: VeriBlock Proof-of-Proof 51%-Attack Prevention
+category: core
+status: Draft
+type: Standards Track
+discussions-to: https://github.com/max-sanchez/ECIPs-1/issues/?
```
Comment (q9f) [inline ECIP-VeriBlock-PoP-51%-Protection.md:8]: Uh, this requires some valid link. Easiest would be to create an issue here on the repository.

**Comment by max-sanchez** [comment]
@q9f Moved and renamed the spec and it's correlating assets, and added a valid discussions link to a new issue. Let me know if there are any other formatting changes needed.

**Review by TheEnthusiasticAs — APPROVED** [review]

**Review by q9f — APPROVED** [review]

**Pull request merged**